azure-pipelines.yml

```diff
@@ -226,6 +226,7 @@ jobs:
 
   - task: AzureArtifacts.manifest-generator-task.manifest-generator-task.ManifestGeneratorTask@0
     displayName: 'SBOM'
+    condition: eq(variables['Build.Reason'], 'BatchedCI')
```

Comment:
This condition does not trigger the task from `IndividualCI`. See https://learn.microsoft.com/en-us/azure/devops/pipelines/build/variables?view=azure-devops&tabs=yaml#build-variables-devops-services